app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -123,12 +138,15 @@ class StateFragmentPresenter @Inject constructor(
       adapter = stateRecyclerViewAdapter
     }
     recyclerViewAdapter = stateRecyclerViewAdapter
-    viewModel = getStateViewModel()
     binding.let {
       it.lifecycleOwner = fragment
       it.viewModel = this.viewModel
     }
 
+    // Initialize Audio Player
```

Comment:
Should we only do this if AudioFragment isn't already added (such as after a configuration change)?